src/components/Signin.jsx

```diff
@@ -0,0 +1,268 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import styled from 'styled-components';
+
+const Content = styled.div`
+  width: 70%;
+`;
+
+const Button = styled.button`
+  width:${props => props.width};
+  margin-left:${props=>props.marginLeft};
+`;
+
+const AuthLinkWrapper = styled.div`
+  margin:40px;
+  h2 { 
+    width:100%; 
+    text-align:center; 
+    border-bottom: 5px solid gray; 
+    line-height:0.1em; 
+    margin:10px 0 20px;
+  } 
+  h2 span { 
+    background:#fff; 
+    padding:0 10px; 
+  }
+
+`;
+
+const DisplayForLinks = styled.div`
+  display: inline;
+  list-style:none;
+  text-align: center;
+  width: 100%;
+  margin: auto;
+  padding: 0 50px;
+  margin-top: 6px;
+  justify-content: space-around;
+  li{
+    margin:10px;
+  }
+  a{
+      width: 90%;
+  }
+  @media screen and (max-width: 650px) {
+    .col {
+      width: 100%;
+      margin-top: 0;
+    }
+  }
+`;
+
+const AuthLink = styled.a`
+  background-color:${props => props.bgColor} ;
+  color: ${props => props.color};
+  &:hover{
+    background-color:${props => props.hoverBgColor};
+    color:${props => props.hoverColor};
+  }
+  i{
+    color:${props => props.IconColor};
+    padding-right:${props => props.paddingRight};
+    background-color:${props => props.IconBgColor};
+  }
+  i &:hover{ 
+    color:${props => props.hoverIconColor};
+  }
```

Comment:
add span property -> flex-basis: 160px

Reply:
ok

Reply:
why do you want to add this property, when even thought they are working fine, i checked , the links are totally responsive, and the text doesnt flow out aslo., then why do you want to add ?